File: C_programming/mario.c
#include <cs50.h>
#include <stdio.h>

int main(void)
{
    // Prompt pyramid height
    int n;

    // Accept height between 1 and 8
    do
    {
        n = get_int("Height: ");
    }
    while (n < 1 || n > 8);

    // For height
    for (int i = 1; i <= n; i++)
    {
        // For columns
        int s = n - i;
        //  left space
        for (int j = 0; j < s; j++)
        {
            printf(" ");
        }
        // Left block
        for (int j = 0; j < i; j++)
        {
            printf("#");
        }

        // Middle space
        for (int j = 0; j < 2; j++)
        {
            printf(" ");
        }

        // Right block
        for (int j = 0; j < i; j++)
        {
            printf("#");
        }
        // Move to newline at the end of each column
        printf("\n");
    }

}

File: mario.py
# TODO
# Prompt user for height
while True:
    try:
        height = int(input("Height: "))
        if height > 0 and height < 9:
            break
    except:
        continue

# Print the height
for i in range(1, height+1):
    s = height - i
    # Left space
    for j in range(s):
        print(" ", end="")
    # Left blocks
    for j in range(i):
        print("#", end="")
    # Space in-between
    print("  ", end="")
    # Right blocks
    for j in range(i):
        print("#", end="")
    print("")